packages/nextra-theme-docs/src/types.ts

```diff
@@ -1,19 +1,14 @@
 import { FC, ReactNode } from 'react'
+import { PageOpts } from 'nextra'
 import { ThemeProviderProps } from 'next-themes/dist/types'
 
 export interface DocsThemeConfig {
   projectLink?: string
   github?: string
   projectLinkIcon?: ReactNode | FC
   docsRepositoryBase?: string
-  titleSuffix?:
-    | string
-    // Can't be React component, otherwise will get Warning: A title element received an array with more than 1 element as children.
-    | ((props: {
-        config: DocsThemeConfig
-        title: string
-        meta: Meta
```

Comment:
removed these props, since they can be retrieved via `useConfig` hook

Reply:
This definitely makes it 100x cleaner!